app/src/main/java/com/example/habitsmasher/LoginFragment.java

```diff
@@ -147,11 +147,11 @@ public void onSuccess(DocumentSnapshot documentSnapshot) {
 
                             saveUserInformation(user);
 
-                            navigateToFragmentWithAction(R.id.action_navigation_login_to_ProfileFragment);
+                            navigateToFragmentWithAction(R.id.action_navigation_login_to_HomeFragment);
```

Comment:
This change ensures that the **home screen** is displayed after login success instead of the **profile screen**.